Core/Newsletter2Go/Views/backend/newsletter2go/view/main/connect.js

```diff
@@ -14,21 +14,17 @@ Ext.define('Shopware.apps.Newsletter2go.view.Connect', {
         labelWidth: 160,
         anchor: '100%'
     },
-    snippets: {
-        title: '{s name=general/title}Connect to Newsletter2Go{/s}'
-    },
     initComponent: function () {
         var me = this;
 
-        me.title = me.snippets.title;
+        me.title = 'Connect to Newsletter2Go';
```

Comment:
what about the localization for this string?

Reply:
actually localization were already not used in previous version, but I have used same approach as it was.